### PR TITLE
Added necessary info to run the docker

### PR DESCRIPTION
**Pull request by petr-tik**
expose the default flask port to the host machine

run.py not present yet, will be run

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,3 +16,6 @@ ADD https://github.com/FFIG/ffig/archive/master.zip /home/ffig/ffig
 
 # Copy in the content of this repository
 COPY . /home/ffig/rest-api
+WORKDIR /home/ffig/flask
+EXPOSE 5000
+RUN python ./run.py
```
Comment (ajbennieston) [inline Dockerfile]: If it isn't present yet, let's add this line when it is present.

**Review by ajbennieston — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ajbennieston) [reply Dockerfile]: Also, is RUN the right thing? I thought RUN was to issue commands inside the image while building it; it sounds like this is the entry point when starting a container. I think there's a different command for that?

Comment (petr-tik) [reply Dockerfile]: you are right, closed it

Comment (ajbennieston) [reply Dockerfile]: Apologies for any confusion - I was happy with the `WORKDIR` and `EXPOSE` lines; it was only the `RUN` line I wanted to ask about.

**Review by ajbennieston — APPROVED** [review]

**Comment by ajbennieston** [comment]
Good to go.

**Comment by ajbennieston** [comment]
Looks like this repo isn't set up to `merge --squash`...

**Pull request merged**